apps/sharedtest/test/unit/dialer/contacts_test.js

```diff
@@ -80,7 +80,7 @@ suite('dialer/contacts', function() {
   suite('> Contacts Revision', function() {
     test('> Should return revision', function() {
       Contacts.getRevision(function(status) {
-        assert.deepEqual(status, 'fakeRevision');
+        assert.deepEqual(status.test, 'fakeRevision');
```

Comment:
I'm not sure how this could worl ? Maybe we want assert.equals ?

Actually looking at the code I think our mocking code is wrong..